install.sh

```diff
@@ -20,3 +20,8 @@ fi
 pip install -r requirements.txt
 
 . download-api-raml.sh
+
+cd schema/api-raml
```

Comment:
Oh yes, this will not get merged and does not need to. I just wanted a convenient way to check for validation against that RAML branch and the parser + bot-lax-adaptor to uncover issues.